src/directive/rateLimit.ts

```diff
@@ -95,7 +96,7 @@ export const onLimit: RateLimitOnLimit<Context> = (
     case 'createFreeformPost':
     case 'submitExternalLink':
     case 'sharePost':
-      context.rateLimitCouner.add(1, { type: 'createPost' });
+      context.meter && counters?.api?.rateLimit?.add(1, { type: 'createPost' });
```

Comment:
can this be in some utility method which under the hood does `meter &&`, seems clunky to call it each time and also everyones understanding when to call it?

Reply:
Might not need `context.meter &&` when we're optionally chaining it